index.js

```diff
@@ -0,0 +1,36 @@
+var express = require("express");
+var hbs = require("express-handlebars");
+var compliments = require("./models/compliments");
+var colors = require("./models/colors");
+
+var app = express();
+
+app.set("view engine", "hbs");
+
+app.engine(".hbs", hbs({
+  extname:        ".hbs",
+  partialsDir:    "views/",
+  layoutsDir:     "views/",
+  defaultLayout:  "layout-main"
+}));
+
+app.use("/assets", express.static("public"));
+
+function randomItem(model){
```

Comment:
clever! If this were a bigger app, would recommend moving some of this code and functionality into its own front-end model...